code/datums/gamemodes/crash.dm

```diff
@@ -194,22 +196,11 @@
 	if(stored_larva)
 		return //No need for respawns
 	var/num_xenos = xeno_hive.get_total_xeno_number() + stored_larva
-	if(!num_xenos)
+	if(num_xenos < 2)
 		xeno_job.add_job_positions(1)
```

Comment:
Shouldn't this add job positions adjust according to `num_xenos`? `2 - num_xenos`, perhaps?